### ISSUE TITLE
Updating Drupal Core to 9.4.3 causes File Uploads to Fail

### ISSUE THREAD

**Issue by seth-shaw-asu** [asulibraries/islandora-repo#548]
This morning we applied security updates for Drupal and Islandora as well as an update for the ArchivesSpace integration. Later this morning Meredith reported that adding files to media were causing errors:
![image](https://user-images.githubusercontent.com/108362375/181628034-aab571bd-5eb9-47e0-a657-29af8c66444b.png)
This also shows up in [watchdog](https://prism.lib.asu.edu/admin/reports/dblog/event/6242194) with the following message:
```
Warning: file_get_contents(core/assets/vendor/jquery.ui/ui/jquery-1-7-min.js): Failed to open stream: No such file or directory in Drupal\Core\Asset\JsCollectionOptimizer->optimize() (line 123 of /var/www/html/drupal/web/core/lib/Drupal/Core/Asset/JsCollectionOptimizer.php)

#0 /var/www/html/drupal/web/core/includes/bootstrap.inc(347): _drupal_error_handler_real()
#1 [internal function]: _drupal_error_handler()
#2 /var/www/html/drupal/web/core/lib/Drupal/Core/Asset/JsCollectionOptimizer.php(123): file_get_contents()
#3 /var/www/html/drupal/web/core/lib/Drupal/Core/Asset/AssetResolver.php(298): Drupal\Core\Asset\JsCollectionOptimizer->optimize()
#4 /var/www/html/drupal/web/core/lib/Drupal/Core/Render/HtmlResponseAttachmentsProcessor.php(319): Drupal\Core\Asset\AssetResolver->getJsAssets()
#5 /var/www/html/drupal/web/core/lib/Drupal/Core/Render/HtmlResponseAttachmentsProcessor.php(158): Drupal\Core\Render\HtmlResponseAttachmentsProcessor->processAssetLibraries()
#6 /var/www/html/drupal/web/core/modules/big_pipe/src/Render/BigPipeResponseAttachmentsProcessor.php(94): Drupal\Core\Render\HtmlResponseAttachmentsProcessor->processAttachments()
#7 /var/www/html/drupal/web/core/lib/Drupal/Core/EventSubscriber/HtmlResponseSubscriber.php(45): Drupal\big_pipe\Render\BigPipeResponseAttachmentsProcessor->processAttachments()
#8 [internal function]: Drupal\Core\EventSubscriber\HtmlResponseSubscriber->onRespond()
#9 /var/www/html/drupal/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php(142): call_user_func()
#10 /var/www/html/drupal/vendor/symfony/http-kernel/HttpKernel.php(191): Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
...trimmed...
```

Oddly, I can't find any reference to jquery 1.7  as being minified, so I don't know why the optimizer thought this file should exist in the first place. I attempted, as a temporary solution, to manually download this particular file to the indicated directory. This worked to get Meredith past the initial error, but another error appeared:
![image (1)](https://user-images.githubusercontent.com/108362375/181629483-07f4af6e-86af-4ec4-88a6-9597a9242dfd.png)
This particular error is not showing up in watchdog, so it appears to be purely a JavaScript issue.

Meredith further explained that this issue was known to happen with more recent versions of Drupal, which is why we hadn't done so yet. (News to me.) She directed me to @KManegold who was unavailable and so directed me to @nightowlaz who didn't recall this as a known issue preventing updates.

That is where we currently stand at this moment. Ideally, we can find the conflict to preserve the Drupal security update.

**Comment by seth-shaw-asu** [comment]
Drupal decided that, as of [9.3.10](https://www.drupal.org/project/drupal/releases/9.3.10), they would transition back to providing jQuery and bumped the version from 1.7 to 1.13. Ergo, our bump to Drupal 9.4.4 bumped our installed version of jQuery to 1.13.

This normally wouldn't be a problem, but we had installed some modules (jquery_ui, jquery_ui_touch_punch, and jquery_ui_slider) to fill-ins for the deprecated jQuery 1.7. When jQuery 1.7 disappeared, these modules no longer functioned properly.  I uninstalled the modules in question from the site which _appears_ to have fixed the issue, but I'm waiting on further feedback from the repo-ops channel for verification that I didn't break something else in the process.

_Side note: we_ really _need to get the PRISM staging server working again._
